pkg/controller/control.go

```diff
@@ -714,18 +715,28 @@ func (c *Controller) ReadAt(b []byte, off int64) (int, error) {
 }
 
 func (c *Controller) handleErrorNoLock(err error) error {
+	snapshotDiskAlreadyExist := false
```

Comment:
Suggest moving this to the same layer where the variable is used.

Reply:
Addressed. Thank you.